src/xnnpack/simd/f32-hvx.h

```diff
@@ -146,7 +150,7 @@ static XNN_INLINE xnn_simd_f32_t xnn_loadu_f32(const float* ptr) {
 }
 
 static XNN_INLINE xnn_simd_f32_t xnn_load_f32(const float* ptr) {
-  return *((HVX_Vector*) ptr);
+  return *((HVX_UVector*) ptr);
```

Comment:
This is supposed to be an aligned load, `xnn_loadu_f32` is above and is an unaligned load